File: exercise/examples/fibonacci_iteration.c
#include <stdio.h>
int fibonacci(int);
int main (void)
{
	printf ("**********************\n");
	printf ("Fibonacci Computing Program ....\n");
	printf ("Please enter the term number : ");
	int term ;
	scanf ("%d",&term);
	printf ("%d\n",fibonacci(term));
	printf ("**********************\n");
	return 0;


}
int fibonacci (int term)

{
	int a = 1 ;
	int b = 1 ;
	int i = 2 ;
	int temp ;
	while (i<term)
	{
		temp = a;
		a = b;
		b = temp+b;
		i++;
	
	
	}
	return b;



}

File: exercise/examples/fibonacci_iteration.py
print ("Fibonacci sequences 1 1 2 3 5 8 ...")
"""
a = 1       b = 1
a = 1       b = 2 (a+b)
a = 2       b = 3 (a+b)



"""
term_number = int (raw_input ("Enter the term number :"))
def fibonacci (term_number):
    a = 1
    b = 1
    i = 2
    while(i<term_number):
        temp = b
        b = a+b
        a= temp
        i+=1
    return b
print (fibonacci (term_number))
